cor_surrogates/code/renv.lock

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could you remove the `renv` files that have shown up in the `cor_surrogates` directory?

Reply:
Done